src/python/pants/bin/daemon_pants_runner.py

```diff
@@ -256,26 +255,36 @@ def run(self):
         ), hermetic_environment_as(
             **self._env
         ), encapsulated_global_logger():
+
+            exit_code = PANTS_SUCCEEDED_EXIT_CODE
             try:
                 # Clean global state.
                 clean_global_runtime_state(reset_subsystem=True)
 
-                options, _, options_bootstrapper = LocalPantsRunner.parse_options(
+                options, build_root, options_bootstrapper = LocalPantsRunner.parse_options(
```

Comment:
```suggestion
                options, _build_root, options_bootstrapper = LocalPantsRunner.parse_options(
```
I'll often do this to avoid lint errors but if I want to remind myself what the tuple element actually is.